packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -150,6 +152,7 @@ const requestRemoteNode = (url, headers, tmpFilename, httpOpts, attempt = 1) =>
       }
       timeout = setTimeout(handleTimeout, STALL_TIMEOUT)
     }
+    let contentLength
```

Comment:
any reason to hoist the declaration?

Reply:
No, I think that's a side effect of some debugging for which I moved it. Removing the hoisting.